cmd/api/app/routes/node/handler.go

```diff
@@ -0,0 +1,25 @@
+package node
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/karmada-io/dashboard/cmd/api/app/router"
+	"github.com/karmada-io/dashboard/cmd/api/app/types/common"
+	"github.com/karmada-io/dashboard/pkg/client"
+	"github.com/karmada-io/dashboard/pkg/resource/node"
+)
+
+func handleGetClusterNode(c *gin.Context) {
+	karmadaClient := client.InClusterClientForKarmadactlApiServer(c.Param("clustername"))
+	dataSelect := common.ParseDataSelectPathParameter(c)
+	result, err := node.GetNodeList(karmadaClient, dataSelect)
+	if err != nil {
+		common.Fail(c, err)
+		return
+	}
+	common.Success(c, result)
+}
+
+func init() {
+	r := router.V1()
+	r.GET("/node/:clustername", handleGetClusterNode)
```

Comment:
and we should add prefix for member cluster resource, currently resources name with / prefix represent the resource in the control plane, which means we get the resource from karmada apiserver. for member cluster's resources, need prefix like `/member/:clustername`, so the route should be corrected as `/member/:clustername/node`, any suggestions?